### PR TITLE
Merge mainnet ttd and bellatrix values

### PR DESCRIPTION
**Pull request by djrtwo**
* `BELLATRIX_FORK_EPOCH`: 144896  (Sept 6, 2022, 11:34:47am UTC)
* `TERMINAL_TOTAL_DIFFICULTY`: 58750000000000000000000 (Estimated: Sept 15, 2022)

### CODE REVIEW

**Review by terencechain — APPROVED** [review]

**Review by hwwhww — APPROVED** [review]
I verified it with my faith in https://slots.symphonious.net/, https://notes.ethereum.org/@MarioHavel/merge-ttd, and https://bordel.wtf/

Kudos to @ajsutton and @taxmeifyoucan!

**Review by ajsutton — APPROVED** [review]

**Review by mkalinin — APPROVED** [review]

**Review by g11tech — APPROVED** [review]

**Review by dapplion — APPROVED** [review]
:panda_face:

**Review by tuyennhv — APPROVED** [review]

File: configs/minimal.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ CONFIG_NAME: 'minimal'
 
 # Transition
 # ---------------------------------------------------------------
-# TBD, 2**256-2**10 is a placeholder
+# 2**256-2**10 for fake minimal network
```
Comment (rolfyone) [inline configs/minimal.yaml:15]: maybe use `testing` instead of `fake`?

Comment (rolfyone) [inline configs/minimal.yaml:15]:
```suggestion
# 2**256-2**10 for testing minimal network
```

**Pull request merged**